Makefile

```
@@ -192,4 +197,4 @@ clean:
 	cargo clean --manifest-path examples/rocksstore/Cargo.toml
 	cargo clean --manifest-path examples/multi-raft-kv/Cargo.toml
 
-.PHONY: test fmt lint clean doc guide
+.PHONY: test fmt lint clean doc guide detsim
```

checkmake (0.2.2)

[warning] Missing required phony target "all"

(minphony)